test/BUILD

```diff
@@ -270,6 +271,27 @@ stardoc_test(
     input_file = "testdata/config_apis_test/input.bzl",
 )
 
+bzl_library(
+    name = "table_of_contents_test_deps",
+    srcs = [
+        "testdata/aspect_test/input.bzl",
+        "testdata/function_basic_test/input.bzl",
+        "testdata/module_extension_test/input.bzl",
+        "testdata/provider_basic_test/input.bzl",
+        "testdata/repo_rules_test/input.bzl",
+        "testdata/simple_test/input.bzl",
+    ],
+)
+
+stardoc_test(
+    name = "table_of_contents_test",
```

Comment:
I could set all the other templates beside the ToC template to an empty template to make this test more resilient to changes in the other rules used to get a ToC. Or I could create a separate set of rules / functions / module extensions / providers / repo rules specifically for this test.

Reply:
Sounds good